### PR TITLE
[4.3] docs(window): Revamp Overview docs page

### PR DESCRIPTION
**Pull request by xristianstefanov**
Related to: https://github.com/telerik/blazor/issues/6304.

### CODE REVIEW

File: components/window/overview.md [hunk 1]
```diff
@@ -53,18 +46,27 @@ The Window component displays a popup window, which shows users custom content.
 
 ## Size
 
-The Window can occupy a predefined size on the screen, or expand automatically, based on the content. By default, users can resize the Window. Learn more about the [Window features, related to size and resizing]({%slug components/window/size%}).
+The Window can occupy a predefined size on the screen, or expand automatically, based on the content. By default, users can resize the Window. [Read more about the Blazor Window size...]({%slug components/window/size%}).
```
Comment (yordan-mitev) [inline components/window/overview.md:49]:
```suggestion
The Window can occupy a predefined size on the screen or expand automatically based on the content. By default, users can resize the Window. [Read more about the Blazor Window size...]({%slug components/window/size%})
```

File: components/window/overview.md [hunk 2]
```diff
@@ -53,18 +46,27 @@ The Window component displays a popup window, which shows users custom content.
...
+
+## Position
+
+The component position can be offset with the `Top` and `Left` parameters. [Read more about the Blazor Window position...]({%slug components/window/position%}).
```
Comment (yordan-mitev) [inline components/window/overview.md:53]:
```suggestion
You can offset the position of the component with the `Top` and `Left` parameters. [Read more about the Blazor Window position...]({%slug components/window/position%})
```

File: components/window/overview.md [hunk 3]
```diff
@@ -53,18 +46,27 @@ The Window component displays a popup window, which shows users custom content.
...
 
+## Modal
 
-## Responsiveness
+The Window can be modal and prevent interacting with the rest of the page until it closes. [Read more about the Blazor Window Modal...]({%slug components/window/modal%}).
 
-The Window component can be responsive when the browser window size changes. Here is an [example how to achieve responsive Window behavior]({%slug window-kb-responsive%}). One way is to use the `Width` and `Height` parameters of the Window. Another option is to apply CSS styles.
+## Events
 
+The [Window component fires events for visibility, state, size and position changes]({%slug window-events%}). Use them to respond to user actions.
```
Comment (yordan-mitev) [inline components/window/overview.md:69]:
```suggestion
The Window component fires events for visibility, state, size, and position changes. Use these events to respond to user actions. [Read more about the Blazor Window Events...]({%slug window-events%})
```

File: components/window/overview.md [hunk 4]
```diff
@@ -53,18 +46,27 @@ The Window component displays a popup window, which shows users custom content.
...
+The component position can be offset with the `Top` and `Left` parameters. [Read more about the Blazor Window position...]({%slug components/window/position%}).
 
+## Actions
+
+You can maximize, minimize, or close the Window through the action buttons in its titlebar. [Read more about the Blazor Window action buttons...]({%slug components/window/actions%}).
 
 ## Dragging
 
-By default, users can move the Window on the page by dragging its titlebar. Learn more about how to use the [Window's `Draggable` feature]({%slug window-draggable%}).
+You can move the Window on the page by dragging its titlebar. [Read more about the Blazor Window dragging option...]({%slug window-draggable%}).
```
Comment (yordan-mitev) [inline components/window/overview.md:61]:
```suggestion
You can move the Window on the page by dragging its titlebar. [Read more about the Blazor Window dragging option...]({%slug window-draggable%})
```

File: components/window/overview.md [hunk 5]
```diff
@@ -53,18 +46,27 @@ The Window component displays a popup window, which shows users custom content.
...
-The Window can occupy a predefined size on the screen, or expand automatically, based on the content. By default, users can resize the Window. Learn more about the [Window features, related to size and resizing]({%slug components/window/size%}).
+The Window can occupy a predefined size on the screen, or expand automatically, based on the content. By default, users can resize the Window. [Read more about the Blazor Window size...]({%slug components/window/size%}).
+
+## Position
+
+The component position can be offset with the `Top` and `Left` parameters. [Read more about the Blazor Window position...]({%slug components/window/position%}).
 
+## Actions
+
+You can maximize, minimize, or close the Window through the action buttons in its titlebar. [Read more about the Blazor Window action buttons...]({%slug components/window/actions%}).
```
Comment (yordan-mitev) [inline components/window/overview.md:57]:
```suggestion
You can maximize, minimize, or close the Window through the action buttons in its titlebar. [Read more about the Blazor Window action buttons...]({%slug components/window/actions%})
```

File: components/window/overview.md (hunk 2 above)
Comment (xristianstefanov) [reply components/window/overview.md]: Fixed.

File: components/window/overview.md (hunk 3 above)
Comment (xristianstefanov) [reply components/window/overview.md]: Fixed.

File: components/window/overview.md [hunk 6]
```diff
@@ -53,18 +46,27 @@ The Window component displays a popup window, which shows users custom content.
...
 
 ## Dragging
 
-By default, users can move the Window on the page by dragging its titlebar. Learn more about how to use the [Window's `Draggable` feature]({%slug window-draggable%}).
+You can move the Window on the page by dragging its titlebar. [Read more about the Blazor Window dragging option...]({%slug window-draggable%}).
 
+## Modal
 
-## Responsiveness
+The Window can be modal and prevent interacting with the rest of the page until it closes. [Read more about the Blazor Window Modal...]({%slug components/window/modal%}).
```
[the remark this answers is not in the document]
Comment (xristianstefanov) [reply components/window/overview.md]: Fixed.

File: components/window/overview.md (hunk 4 above)
Comment (xristianstefanov) [reply components/window/overview.md]: Fixed.

File: components/window/overview.md (hunk 5 above)
Comment (xristianstefanov) [reply components/window/overview.md]: Fixed.

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**